### PR TITLE
Document refresh token 'retrieve by id' API

### PR DESCRIPTION
**Pull request by mooreds**
Added in support for the refresh token being retrieved by Id. It wasn't documented previously, but was part of the client libraries, so should have been.

This fixes:  https://github.com/FusionAuth/fusionauth-issues/issues/1724

### CODE REVIEW

**Review by spwitt — APPROVED** [review]

File: site/docs/src/json/jwt/refresh-token-get-response.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (robotdan) [inline site/docs/src/json/jwt/refresh-token-get-response.json:1]: Can you format this file? I think our default formatting rules will bring this curly brace up to the previous line

File: site/docs/v1/tech/apis/_refresh-token-response-body-base.adoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+An array of Refresh Tokens
+endif::[]
+
+[field]#{base_field_name}.applicationId# [type]#[String]#::
+The Id of the Application for which this token is authorized to be exchanged for a new Access Token.
+
+[field]#{base_field_name}.device# [type]#[String]# [deprecated]#Deprecated#::
+A unique device identifier to represent the device for which this Refresh Token was issued.
++
+[deprecated]#Removed in version 1.5.0# This parameter will no longer be returned in the response body.
```
Comment (robotdan) [inline site/docs/v1/tech/apis/_refresh-token-response-body-base.adoc:13]: Up to you.. but for some of these really old deprecation warnings, I'm ok with dropping them. 1.5.0 was released in March of 2019. ( https://fusionauth.io/docs/v1/tech/archive/release-notes#version-1-5-0 ).

File: site/docs/v1/tech/apis/_refresh-token-response-body-base.adoc [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+[field]#{base_field_name}.insertInstant# [type]#[Long]#::
+The link:/docs/v1/tech/reference/data-types#instants[instant] this Refresh Token was issued.
+
+[field]#{base_field_name}.id# [type]#[Long]#::
+The Id of the Refresh Token.
```
Comment (robotdan) [inline site/docs/v1/tech/apis/_refresh-token-response-body-base.adoc:19]: Isn't this a `UUID`?

File: site/docs/v1/tech/apis/jwt.adoc [hunk 4]
```diff
@@ -340,6 +340,16 @@ The [field]#jwtConfiguration.refreshTokenUsagePolicy# is configured at the tenan
 
 === Request
 
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve a Single Refresh Token
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/jwt/refresh/``\{tokenId\}``#
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/apis/jwt.adoc:348]: Yes, thanks!

File: site/docs/src/json/jwt/refresh-token-get-response.json (hunk 1 above)
Comment (mooreds) [reply site/docs/src/json/jwt/refresh-token-get-response.json:1]: I pulled the curly brace up. I don't have intellij so can't use the formatting, unfortunately.

File: site/docs/v1/tech/apis/_refresh-token-response-body-base.adoc (hunk 2 above)
Comment (mooreds) [reply site/docs/v1/tech/apis/_refresh-token-response-body-base.adoc:13]: I think 30 versions ago is fine to drop. Will add that to the devdocs guidelines.

File: site/docs/v1/tech/apis/_refresh-token-response-body-base.adoc (hunk 3 above)
Comment (mooreds) [reply site/docs/v1/tech/apis/_refresh-token-response-body-base.adoc:19]: Indeed.

**Review by robotdan — COMMENTED** [review]
LGTM

**Pull request merged**